laravel-app/database/migrations/2023_09_20_011042_create_users_information.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('users_information', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id')->index()->nullable();
```

Comment:
Why user_id column is nullable? so there's a possibility that you have user information data but no related user on it?

Reply:
I'll change it po, the stackoverflow reference that I'm looking has a nullable() method and it turns out that it has a possibility of having data but no related user which is not suitable for this project.